korman/exporter/mesh.py

```diff
@@ -267,22 +288,17 @@ def _export_geometry(self, bo, mesh, materials, geospans):
                         geoVertex.normal = hsVector3(*tessface.normal)
 
                     geoVertex.color = hsColor32(*vertex_color)
-                    geoVertex.uvs = [hsVector3(uv[0], 1.0 - uv[1], 0.0) for uv in uvws]
+                    uvs = list([hsVector3(uv[0], 1.0 - uv[1], 0.0) for uv in uvws])
+                    if bumpmap is not None:
+                        uvs.append(dPosDu)
+                        uvs.append(dPosDv)
```

Comment:
This is slightly different than the original code: In the original, we loop through and each vertex can have multiple UV gradients that are summed and then normalized. In this code, you're just appending already-normalized UV gradients to a list.